helpers.ts

```diff
@@ -163,6 +163,15 @@ export function encodeAltitude(alt: number): string {
     return alt.toFixed(1) + ",M";
 }
 
+// Some encodings don't want the unit
+export function encodeAltitudeNoUnits(alt: number): string {
+    if (alt === undefined) {
+        return ",";
```

Comment:
Since the result of this method will be joined with commas, this should be:

```suggestion
        return "";
```